test/integration/services/storage/storage-manger.spec.ts

```diff
@@ -81,7 +81,7 @@ describe('Storage service', function () {
         const offer = offers[0]
         expect(offer.peerId).to.be.eql(app.peerId?.id as string)
         expect(offer.totalCapacity.toString()).to.be.eql(offerData.totalCapacity)
-        expect(offer.provider).to.be.eql(app.providerAddress)
+        expect(offer.provider).to.be.eql(app.providerAddress.toLowerCase())
```

Comment:
Lets lower case it in the `app` itself and it does not have to be then done in tests almost at all.

Reply:
Yes, that's true. But, from the other side, this will guarantee that we make lower casing ow addresses

Reply:
Hmm good point, alright lets keep it like this then!